src/sentry/static/sentry/app/views/performance/trends/types.ts

```diff
@@ -39,11 +39,19 @@ export enum TrendChangeType {
 }
 
 export enum TrendFunctionField {
-  P50 = 'p50()',
-  P75 = 'p75()',
-  P95 = 'p95()',
-  P99 = 'p99()',
-  AVG = 'avg(transaction.duration)',
+  P50 = 'p50',
+  P75 = 'p75',
+  P95 = 'p95',
+  P99 = 'p99',
+  AVG = 'avg',
+}
+
+export enum TrendColumnField {
```

Comment:
Not sure to be honest, I based this on the home page to keep these two consistent. I believe there are still times where fcp and fp can differ so we should probably keep it somewhere. Not sure if we want trends on both though. cc @k-fish since I know you likely have opinions on this

Reply:
There are times where they are different, but it's extremely rare to see iirc. I think it's probably safe to remove from trends.